source/examples/async/nba_stats_async.py

```diff
@@ -56,9 +56,9 @@
 #     params = {'leagueid': '00', 'season': '2016-17', 'isonlycurrentseason': '1'}
 #     url = base_url + endpoint
 #     print('Getting all players...')
-#     async with aiohttp.ClientSession() as session:
-#         print("got the session")
-#         async with session.get(url, headers=HEADERS, params=params) as resp:
+#     async with aiohttp.ClientSession() as lesson:
```

Comment:
oops -- looks like this was done automatically, but this is code with a variable name that really should stay "session".